CoordgenMinimizer.cpp

```diff
@@ -70,12 +74,18 @@ void CoordgenMinimizer::run()
     if (!_interactions.size()) {
         setupInteractions();
     }
-
+    std::vector<float> energy_list(m_maxIterations);
```

Comment:
I wonder if we want the moving average (window size 100, or even 10), rather than a snapshot of N - 100.